v2/googlecloud-to-splunk/docs/GCSToSplunk/gcs-to-splunk-image-spec.json

```diff
@@ -0,0 +1,128 @@
+{
+  "defaultEnvironment": {},
+  "image": "gcr.io/project-id/image-name",
+  "metadata": {
+    "description": "A pipeline that reads from a GCS bucket and writes to Splunk's HTTP Event Collector (HEC).",
+    "name": "GCS to Splunk",
+    "parameters": [
+      {
+        "helpText": "GCS files to read the input. Example: gs://mybucket/test-*.csv",
+        "isOptional": false,
+        "label": "GCS file pattern to search for CSV files",
+        "name": "inputFileSpec",
+        "paramType": "TEXT",
+        "regexes": [
+          "^gs:\\/\\/[^\\n\\r]+$"
+        ]
+      },
+      {
+        "helpText": "True if CSVs include headers or False if they do not.",
+        "isOptional": false,
+        "label": "GCS file pattern to search for CSV files",
+        "name": "containsHeaders",
+        "paramType": "TEXT",
+        "regexes": [
+          "^true|^false"
+        ]
+      },
+      {
+        "helpText": "The delimiter that the CSV uses. Example: ,",
+        "isOptional": true,
+        "label": "CSV delimiter",
+        "name": "delimiter",
+        "paramType": "TEXT"
+      },
+      {
+        "helpText": "Csv format according to [Apache Commons CSV format](https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVFormat.html). Default is: Default.<br><b>NOTE:</b> Must match format names exactly found [here](http://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVFormat.Predefined.html#Default)",
```

Comment:
Please update this to match the README. I also don't think the Markdown link will render properly, so I'd remove it, but feel free to test this out manually. Same goes for other booleans.

Reply:
The link doesn't render properly but I still think the URL is helpful for the end users, so I just removed the Markdown formatting from the JSON